docker-compose.yml

```diff
@@ -0,0 +1,48 @@
+version: "3"
+services:
+
+  lisk-explorer:
+    image: lisk/explorer:${EXPLORER_TAG}
+    depends_on:
+      - explorer-cache
+      - freegeoip
+    ports:
+      - "127.0.0.1:6040:6040"
+    networks:
+      - lisk-explorer
+      - localhost
+    restart: on-failure
+    environment:
+      - LISK_HOST=${LISK_HOST}
```

Comment:
Variables should be set here by the end user -- we might want to add a placeholder instead -- but not in `.env` and definitely not in the `Dockerfile`.